### PR TITLE
Remove multi-cluster apps references

### LINKED ISSUE #734: Restructure pages related to deploying apps
Re: https://github.com/rancher/rancher-docs/pull/714

The pages in this section of the docs repeat information that also appears under Fleet Gitops at Scale. This is bad for maintenance, as the pages might drift out of sync. We should pick one section or the other. 

The PR referenced above had nothing to do with restructuring. It simply uncovered the problem. In the thread, the proposed the following:

>- For v2.7:
>   - Remove the current [Deploying Applications index page](https://ranchermanager.docs.rancher.com/pages-for-subheaders/deploy-apps-across-clusters) and have the user see [fleet-git-ops-at-scale](https://ranchermanager.docs.rancher.com/pages-for-subheaders/fleet-gitops-at-scale) in that section, in addition to [fleet-git-ops-at-scale](https://ranchermanager.docs.rancher.com/pages-for-subheaders/fleet-gitops-at-scale)'s current location under Rancher Integrations. 
>    - Don't mention the multi-cluster feature as an option, it's been removed. Maybe add a note that it's available in earlier Rancher versions.
>   - Remove both of the current subpages under /deploy-apps, and redirect their paths to [fleet-git-ops-at-scale](https://ranchermanager.docs.rancher.com/pages-for-subheaders/fleet-gitops-at-scale).
>- For versioned_docs:
>    - v2.5 and v2.6 we can leave alone since it's still available in the product.
>    - Leave v2.0-2.4 alone. There's no confusion there.
>- Update the note on some pages, "As of Rancher v2.5, multi-cluster apps are deprecated. We now recommend using [Fleet](https://ranchermanager.docs.rancher.com/v2.6/how-to-guides/new-user-guides/deploy-apps-across-clusters/fleet) for deploying apps across clusters," to point to [fleet-git-ops-at-scale](https://ranchermanager.docs.rancher.com/pages-for-subheaders/fleet-gitops-at-scale)

### PR DESCRIPTION
**Pull request by btat**
Fixes #734 

## Reminders

- See the [README](../README.md) for more details on how to work with the Rancher docs.

- Verify if changes pertain to other versions of Rancher. If they do, finalize the edits on one version of the page, then apply the edits to the other versions.

- If the pull request is dependent on an upcoming release, make sure to target the release branch instead of `main`.

### CODE REVIEW

File: docs/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: Continuous Delivery
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery"/>
 </head>
 
-[Fleet](../../../how-to-guides/new-user-guides/deploy-apps-across-clusters/fleet.md) comes preinstalled in Rancher can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
+[Continuous Delivery with Fleet](../../../integrations-in-rancher/fleet/fleet.md) comes preinstalled in Rancher can't be fully disabled. comes preinstalled in Rancher can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
```
Comment (martyav) [inline docs/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery.md:9]:
```suggestion
[Continuous Delivery with Fleet](../../../integrations-in-rancher/fleet/fleet.md) comes preinstalled in Rancher and can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
```

File: versioned_docs/version-2.7/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ title: Continuous Delivery
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery"/>
 </head>
 
-[Fleet](../../../how-to-guides/new-user-guides/deploy-apps-across-clusters/fleet.md) comes preinstalled in Rancher can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
+[Continuous Delivery with Fleet](../../../integrations-in-rancher/fleet-gitops-at-scale/fleet-gitops-at-scale.md) comes preinstalled in Rancher can't be fully disabled. comes preinstalled in Rancher can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
```
Comment (martyav) [inline versioned_docs/version-2.7/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery.md:9]:
```suggestion
[Continuous Delivery with Fleet](../../../integrations-in-rancher/fleet-gitops-at-scale/fleet-gitops-at-scale.md) comes preinstalled in Rancher and can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
```

File: versioned_docs/version-2.8/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery.md [hunk 3]
```diff
@@ -6,7 +6,7 @@ title: Continuous Delivery
   <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery"/>
 </head>
 
-[Fleet](../../../how-to-guides/new-user-guides/deploy-apps-across-clusters/fleet.md) comes preinstalled in Rancher can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
+[Continuous Delivery with Fleet](../../../integrations-in-rancher/fleet/fleet.md) comes preinstalled in Rancher can't be fully disabled. comes preinstalled in Rancher can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
```
Comment (martyav) [inline versioned_docs/version-2.8/how-to-guides/advanced-user-guides/enable-experimental-features/continuous-delivery.md:9]:
```suggestion
[Continuous Delivery with Fleet](../../../integrations-in-rancher/fleet/fleet.md) comes preinstalled in Rancher and can't be fully disabled. However, the Fleet feature for GitOps continuous delivery may be disabled using the `continuous-delivery` feature flag.
```

**Review by martyav — CHANGES_REQUESTED** [review]
Just fixing a copy and paste error

**Review by sunilarjun — APPROVED** [review]
Smoke tested changes/links and all looks good from my side, thanks Billy!

**Review by martyav — APPROVED** [review]

**Pull request merged**